pages/homePage.js

```diff
@@ -0,0 +1,37 @@
+const { Page } = require('./page');
+
+const loggedAs = '#loggedas';
```

Comment:
@karenn32 There is no need for the constant with the locator, while it used only ib the single method🗿